vscode/src/telemetry.ts

```diff
@@ -2,6 +2,8 @@ import * as vscode from "vscode";
 import TelemetryReporter from "@vscode/extension-telemetry";
 import { log } from "qsharp-lang";
 
+/// <reference types="user-agent-data-types" />
```

Comment:
This should go just after the license header, but just noticed this file forget to add one. Please add. (First  4 lines should look similar to https://github.com/microsoft/qsharp/blob/main/vscode/src/webview/webview.tsx - but with the different package name).